proto/opamp.proto

```diff
@@ -635,7 +649,10 @@ enum AgentCapabilities {
     AgentCapabilities_ReportsHealth                  = 0x00000800;
     // The Agent will report RemoteConfig status via AgentToServer.remote_config_status field.
     AgentCapabilities_ReportsRemoteConfig            = 0x00001000;
-
+    // The Agent will report heartbeats on a default interval of 30s.
```

Comment:
Should we specify that 30 seconds is the default interval? Won't that vary by client implementation?

Reply:
In this case we are saying that by default a 30s interval should be used for heartbeating. Some servers may override this default.